chromium_src/chrome/browser/component_updater/tls_deprecation_config_component_installer.cc

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/component_updater/tls_deprecation_config_component_installer.h"
+
+#define ReconfigureAfterNetworkRestart \
+  ReconfigureAfterNetworkRestart_ChromiumImpl
+#include "../../../../../chrome/browser/component_updater/tls_deprecation_config_component_installer.cc"  // NOLINT
+#undef ReconfigureAfterNetworkRestart
+
+#include "base/bind.h"
+#include "chrome/browser/browser_process.h"
+#include "content/public/browser/browser_thread.h"
+#include "services/network/public/proto/tls_deprecation_config.pb.h"
+
+namespace component_updater {
+
+namespace {
+
+std::string LoadEmptyConfig() {
+  base::ScopedBlockingCall scoped_blocking_call(FROM_HERE,
```

Comment:
also re https://github.com/brave/brave-core/pull/6574/files#r486402761 - if something does require IO then ScopedBlockingCall should definitely not be here because that means you only need that on the UI thread